File: app/src/main/java/com/rachelleignacio/listbucket/domain/interactors/RenameListInteractor.java
package com.rachelleignacio.listbucket.domain.interactors;

import com.rachelleignacio.listbucket.domain.interactors.base.Interactor;

/**
 * Created by rachelleignacio on 4/18/17.
 */

public interface RenameListInteractor extends Interactor {
    interface Callback {
        void onListRenamed();
    }
}

File: app/src/main/kotlin/com/rachelleignacio/listbucket/domain/interactors/RenameListInteractor.kt
package com.rachelleignacio.listbucket.domain.interactors

import com.rachelleignacio.listbucket.domain.interactors.base.Interactor

/**
 * Created by rachelleignacio on 8/9/17.
 */
interface RenameListInteractor : Interactor {
    interface Callback {
        fun onListRenamed()
    }
}